modules/gatsby/src/gatsby-node/process-nodes-json.js

```diff
@@ -9,6 +9,7 @@ module.exports.processNewDocsJsonNode = function processNewDocsJsonNode({
 }, docNodes) {
   traverseTableOfContents(node.chapters, docNodes, 1);
   tableOfContents = node;
+  return node;
```

Comment:
Looks like you left code after the return statement... can't imagine the linter being happy with this...

Reply:
ok I took a pass on that file to make it more linter friendly